test/test_google.py

```diff
@@ -90,6 +72,30 @@ def get_csv_file_contents(file_id):
 
 ###############################################
 # Tests
+def test_open_access_dashboard_folder_id():
+    folder_id = google.open_access_dashboard_folder_id()
+    assert folder_id is not None
+
+
+def orcid_dashboard_folder_id():
+    folder_id = google.orcid_dashboard_folder_id()
+    assert folder_id is not None
+
+
+def test_get_file_id():
+    # Confirm the file exists in the shared google drive
+    assert google_file_exists(google_drive_id(), "Test") is True
```

Comment:
How o we know there is going to be a file "Test" in the Google Drive folder?

Reply:
It was in the README, but it needed some extra clarification, which I added (since for this particular test, it's now important that this file be called "Test" and not just be configured with it's ID)